### ISSUE TITLE
Materialize requires quoting column names conflict with keywords

### ISSUE THREAD

**Issue by jrmuizel** [MaterializeInc/materialize#3139]
### What version of Materialize are you using?
materialized v0.2.2 (3700b22a50e843117005420e67e3c43e56bfd219)

### How did you install Materialize?

Homebrew tap

### What was the issue?

Materialize requires quoting column names that conflict with keywords. Postgresql does not have this requirement.

### Is the issue reproducible? If so, please provide reproduction instructions.

Yep:
```
materialize=> select date from testing_view;
ERROR:  Parse error:
select date from testing_view;
            ^^^^
Expected literal string, found: from
```

See also: https://github.com/andygrove/sqlparser-rs/issues/168

**Comment by benesch** [comment]
Hi, thanks for the report, @jrmuizel. We've got an existing issue for exactly this problem, so I'm going to mark this as a duplicate of #2577, but will bump up the priority of getting it fixed.

**Comment by benesch** [comment]
The fix is in #3146, which will likely land this week.
